src/client/rspamdclient.c

```diff
@@ -19,7 +19,7 @@
 #include "libserver/http/http_private.h"
 #include "libserver/protocol_internal.h"
 #include "unix-std.h"
-#include "contrib/zstd/zstd.h"
```

Comment:
Can we add #ifdef/#ifndef here to keep `"contrib/zstd/zstd.h"` if we don't enable this option. Otherwise there is a risk of messign the build up.

Reply:
yes it could be like that, I thought the `INCLUDE_DIRECTORIES(BEFORE "${CMAKE_SOURCE_DIR}/contrib/zstd")` would make it the same apart from changing `"contrib/zstd/error_public.h"` to `"zstd_errors.h"` which actually look the same but only `zstd_errors.h` is in the distributed library .

Reply:
How about now? I couldn’t get it to work without defining a new definition.